Documentation/project-docs/triage.md

```diff
@@ -48,4 +48,25 @@ In particular, we strive to follow these guidelines:
 * **Do** close issues that represent work we don't think makes sense for the current or upcoming release
 * **Do** close issues that provide little or questionable value to APIs that are included with the .NET Framework
 * **Do** favor issues that do not require modifying types that are included with the .NET Framework.
-* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
\ No newline at end of file
+* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
+
+## Porting to .NET Framework
+
+### Constraints for shipping in the .NET Framework
+ 
+Each release of .NET Framework ships to over one billion machines word-wide and installs as an *in-place* update, requiring each fix to meet a high level of quality and compatibility.  Each fix is reviewed extensively to determine which release if any is most appropriate.
+
+Here are some of the factors that are considered:
+
+* **Backward compatibility**. We do not want to break existing applications.
+* **Risk and/or size of the change**. We need to consider whether the change is on a common code path or causes a lot of churn.
+* **Measurable quality of the change**. We need to pay attention to our ability test the change in an complete end-to-end fashion.
+* **Value of the change**. All the risks outlined above need to be balanced against how many customers the change will help or how much the change will help customers.
```

Comment:
Not quite. We tried to say that we need to strike a balance between risk and value. And value is a function of how many customers it helps and/or how much it helps the affected customers.

Let me given an example. There are changes we can make that improve the lives of all .NET developers (let's say we a new convenience API to `string`). And there are changes we can make certain developer's lives easier by, for instance adding better support for `SIMD`. While SIMD is only used by a fraction of our customers, the once that use it really benefit from it, because it can boost the performance of certain algorithms by several factors.

Does this make sense?